src/test/resources/emails/summaryOfFeedbackSessionsOfCourseEmailForStudent.html

```diff
@@ -13,7 +13,7 @@
 <br>&nbsp;&nbsp; <strong>Feedback Session Name:</strong> Closed Session
 <br>&nbsp;&nbsp; <strong>Deadline:</strong> Sat, 01 Jun 2013, 11:59 PM (Passed)
 <br>
-To submit feedback for the above session, please go to this Web address: (Feedback session is closed)
+To submit feedback for the above session, please go to this Web address: <a href="${app.url}/page/studentFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=Closed+Session&key=${regkey.enc}&studentemail=student1InCourse1%40gmail.tmt">${app.url}/page/studentFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=Closed+Session&key=${regkey.enc}&studentemail=student1InCourse1%40gmail.tmt</a>
```

Comment:
This is slightly misleading since the user cannot actually submit/edit feedback at this link anymore. Perhaps change the userEmailTemplateFragment file to say "To view/submit feedback for the above sessions...".